### DISCUSSION TITLE
Anyone able to get this new health icon system to work

### DISCUSSION THREAD

**Discussion by Davesunavailable** [KadeArchive/Kade-Engine#2239] [Q&A]
I know im prob just missing a single line of code or smth but I just cannot seem to get a character to load.

I can get the assets to load and I can get the icon to load but when I go to boot a song, It just crashes without a crash message.

I cant seem to find a tutorial on how to use this new character system so if anyone can link me a tutorial or just tell me when to add to get it to work, that would be fine.

**Comment by NK-523** [comment]
go to freeplaylist.txt and change `Bopeebo:dad:1` to` Bopeebo:(your character's icon name):1`, so if you want GF's icon instead of dad's make it `Bopeebo:gf:1` to test the icon.

**Reply by NK-523** [reply]
if it looks like this: 
![image](https://user-images.githubusercontent.com/77185327/133007683-237e63d2-84d4-4be8-a433-c76dcdd96054.png)
something is wrong with your icon,

**Reply by BrandonHatesAnime** [reply]
In new versions of Kade Engine, you don't need to set the character health icon in HealthIcon.hx.

Instead, go to the folder you have Kade Engine set up in, navigate to assets\preload\images\icons, then create an image (or duplicate an existing one) and rename it "icon-[character].png", with [character] being whatever name you put in characterList.txt.

Edit: I got this on YT. Credit to the person who actually made it https://www.youtube.com/channel/UCbN4A0324-iEVZrpKHHJMaA
